### ISSUE TITLE
Provide exe?

### ISSUE THREAD

**Issue by kanishka-azimi** [jfischoff/tmp-postgres#12]
Hi,

If you are interested, can you provide this as an exe as well?

Thanks

**Comment by jfischoff** [comment]
Yeah I think that is a good idea. I've wanted that myself and I end up using `pg_tmp` but is doesn't have the localhost option which can be a problem.
